content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,117 @@
+# Using CML on GitLab
+
+Here, we'll walk through a tutorial to start using CML on GitLab. For
+simplicity, we'll show the demo in GitLab Actions, but these instructions are
+valid for all supported Continuous Integration (CI) systems (with exceptions as
+noted!).
+
+1. Fork our
+   [example project repository](https://gitlab.com/iterative.ai/example_cml).
+   Click on Fork and select what namespace to fork the project to.
+
+   ![](/img/gitlab_fork_cml_project.png)
+
+   > 💡 If just starting to use GitLab, you may receive a warning at the top of
+   > the page regarding SSH keys. You do not need to use SSH keys for the
+   > tutorial so you can click: "Don't show me again."  
+   > [Click this link for more information on SSH keys.](https://docs.gitlab.com/ee/ssh/)
+
+2. ⚠️ CML in GitLab works exactly as in GitHub except for a few conventions. In
+   Gitlab, to use CML, you must create a variable called a repo_token whose
+   value is a Personal Access Token. To do this:
+
+   a. Click on your Avatar in the upper right side and click on Edit Profile.
+
+   b. Along the left side of the screen go to Access Tokens.
+
+   c. In the "Name" field, type `repo_token` and check boxes to select `api`,
+   `read_repository` and `write_repository`.
+
+   d. Click on the "Create personal access token" button and copy the generated
+   access token.
+
+   ![](/img/personal_access_token.png)
+
+   e. Head back to your forked repository by clicking the Projects tab next to
+   the GitLab logo and select your forked project.
+
+   f. On the left hand side Navigate to **Settings** ➡ **CI/CD** ➡ **Varibles**.
+
+   ![](/img/ci_cd_navigation.png)
```

Comment:
Again, resolution = 💯 but maybe zoom into that corner for this one? Something like:

![image](https://user-images.githubusercontent.com/1477535/108446690-24fece80-7224-11eb-8c42-34cb6d83ef1f.png)

> Maybe a bit taller, up to you